### ISSUE TITLE
The behavior of this function in VB 2015 should be documented in more detail.

### ISSUE THREAD

**Issue by pschwede** [MicrosoftDocs/VBA-Docs#327]
Example of counter-intuitive behavior:
'''VB
MyVar = "459.."     ' Assign value.
MyCheck = IsNumeric(MyVar)      ' Returns True

MyVar = "459.-"     ' Assign value.
MyCheck = IsNumeric(MyVar)      ' Returns True
'''

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1b743c26-e302-d773-621c-149ab06f2eec
* Version Independent ID: 853ca5f5-3378-3412-0db9-92bb1788e07f
* Content: [IsNumeric Function](https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/isnumeric-function)
* Content Source: [Language/Reference/User-Interface-Help/isnumeric-function.md](https://github.com/MicrosoftDocs/VBA-Docs/blob/master/Language/Reference/User-Interface-Help/isnumeric-function.md)
* Product: **office**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by pschwede** [comment]
'''VB
IsNumeric("459.*") ' Returns False
'''

**Comment by tbelliveau** [comment]
Obviously, from the feedback you have received, @Microsoft doesn't care very much about this site!
I wonder if I'll get a response?
